app/helpers/seasonal_helper.rb

```diff
@@ -1,6 +1,10 @@
 module SeasonalHelper
   def christmas(current_time)
-    current_time.month == 12 && current_time.day > 6
+    current_time.month == 12 && current_time.day >= 24 && current_time.day <= 26
+  end
+
+  def december(current_time)
+    current_time.month == 12 && ((current_time.day > 6 && current_time.day < 24) || current_time.day > 26)
```

Comment:
```suggestion
    current_time.month == 12 && current_time.day > 6
```

The return in line 26 allows to simplify this.